File: java/Inheritance/Parcel.java
     public class Parcel
    {
        private int id;
        private String senderName;
        private String senderAddress;
        private String receiverName;
        private String receiverAddress;
        private int weight;
        private int fee;

	void setId(int id){
			this.id = id;
		}
		int getId(){
			return id;
		}
		
		void setSenderName(String sname){
			this.senderName = sname;
		}
		String getSenderName(){
			return senderName;
		}
		
		void setSenderAddress(String saddress){
			this.senderAddress = saddress;
		}
		String getSenderAddress(){
			return senderAddress;
		}
		
		void setReceiverName(String rname){
			this.receiverName = rname;
		}
		String getReceiverName(){
			return receiverName;
		}
		
		void setReceiverAddress(String raddress){
			this.receiverAddress = raddress;
		}
		String getReceiverAddress(){
			return receiverAddress;
		}
		
		void setWeight(int w){
			this.weight = w;
		}
		int getWeight(){
			return weight;
		}
		
		void setFee(int f){
			this.fee = f;
		}
		int getFee(){
			return fee;
		}

    }
  

File: cpp/02 Constructor , Copy Constructor and Case Study/CPP/CPP/Parcel.cpp
#include <iostream>
#include <stdlib.h>
using namespace std;

class Parcel{
	private:
		int Id;
		string senderName;
		string senderAddress;
		string receiverName;
		string receiverAddress;
		int weight;
		int fee;
	public:
		 
		Parcel(){
			senderName = "";
			senderAddress = "";
			receiverName = "";
			receiverAddress = "";
			weight = 0;
			fee = 0;
		}
		  
		void setId(int id){
			Id = id;
		}
		int getId(){
			return Id;
		}
		
		void setSenderName(string sname){
			senderName = sname;
		}
		string getSenderName(){
			return senderName;
		}
		
		void setSenderAddress(string saddress){
			senderAddress = saddress;
		}
		string getSenderAddress(){
			return senderAddress;
		}
		
		void setReceiverName(string rname){
			receiverName = rname;
		}
		string getReceiverName(){
			return receiverName;
		}
		
		void setReceiverAddress(string raddress){
			receiverAddress = raddress;
		}
		string getReceiverAddress(){
			return receiverAddress;
		}
		
		void setWeight(int w){
			weight = w;
		}
		int getWeight(){
			return weight;
		}
		
		void setFee(int f){
			fee = f;
		}
		int getFee(){
			return fee;
		}
		
};

